Directory.Packages.props

```diff
@@ -0,0 +1,90 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  
+  <!-- Run "dotnet list package (dash,dash)outdated" to see the latest versions of each package.-->
```

Comment:
Note that this command is actually `dotnet list package --outdated` but `--` isn't allowed in xml comments